liquibase-core/src/test/java/liquibase/sqlgenerator/core/util/MSSQLUtilTest.java

```diff
@@ -0,0 +1,91 @@
+package liquibase.sqlgenerator.core.util;
+
+import liquibase.sql.Sql;
+import liquibase.structure.core.View;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MSSQLUtilTest {
```

Comment:
Could you please add a few test cases?
1. Where SQL is not upper case, so case sensitivity is also checked
2. Where SET statement is in the same line as usual statement (e.g. INSERT)